source/plugin/configuration/index.rst

```diff
@@ -79,35 +84,18 @@ If you are unsure of what to set the value of ``sharedRoot`` to, consider the fo
     @DefaultConfig(sharedRoot = true)
     private ConfigurationLoader<CommentedConfigurationNode> configManager;
 
+    @Inject
+    @ConfigDir(sharedRoot = false)
+    private File privateConfigDir;
```

Comment:
You should either mention both or only `Path` - the recommended way is using `Path` now.